README.md

```diff
@@ -1,5 +1,4 @@
-fork my repo
-
-1: You will create a branch, with your name, david_liang
-2: in your branch, you will create a new file (name of the file, david.md)
-3: you will list 3~5 ideas about what is frontend development
+create websites
+User interface
+Graphic design
+Programming languages
```

Comment:
yeas, it's fun that you need to bring together different programming languages to make a website and application.